driver/registry_default_registration.go

```diff
@@ -17,9 +17,7 @@ func (m *RegistryDefault) PostRegistrationPrePersistHooks(ctx context.Context, c
 	if err != nil {
 		return nil, err
 	}
-	if credentialsType == identity.CredentialsTypeCodeAuth && m.Config().SelfServiceCodeStrategy(ctx).PasswordlessEnabled {
```

Comment:
This is done here:

https://github.com/ory/kratos/blob/b1628976a0251a0ad84fd2128d1df23f4dff5e99/selfservice/strategy/code/strategy_registration.go#L266-L308

https://github.com/ory/kratos/blob/b1628976a0251a0ad84fd2128d1df23f4dff5e99/selfservice/strategy/code/strategy_login.go#L518-L542